test/unit/data-type.js

```diff
@@ -75,9 +75,322 @@ describe('Data Types', function() {
       const parameter = { value: value, scale: scale };
       parameter.value.nanosecondDelta = nanosecondDelta;
 
-      const buffer = new WritableTrackingBuffer(16);
-      type.writeParameterData(buffer, parameter, { useUTC: false }, () => {});
+      const buffer = new WritableTrackingBuffer(0);
+      type.writeParameterData(buffer, parameter, { useUTC: false }, () => { });
       assert.deepEqual(buffer.data, expectedBuffer);
     }
   });
+
+  it('should writeParameterData bigInt', () => {
+    const type = TYPES.typeByName.BigInt;
+
+    for (const [value, expected] of [
+      [123456789, Buffer.from('0815cd5b0700000000', 'hex')],
+      [null, Buffer.from('00', 'hex')],
+    ]) {
+      const buffer = new WritableTrackingBuffer(0);
+      const parameterValue = { value };
+      type.writeParameterData(buffer, parameterValue, { useUTC: false }, () => { });
+      assert.isTrue(buffer.data.equals(expected));
```

Comment:
I think it would be better to use `assert.deepEqual(buffer.data, expected);` here. 

In case of a failure `assert.isTrue` will only tell you that something was `false` instead of `true`, while `assert.deepEqual` will show the individual bytes, making it a bit easier to understand which bytes were different (and what values they had).

You should always try to use the most specific assertion functions instead of more generic ones.